### PR TITLE
BXMSDOC-4018: Installing and configuring DM/PAM 7.4 in an EAP clustered environment

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+.Prerequisites
+* A Java Virtual Machine (JVM) environment compatible with Java 8.0 or later is installed.
+* A backed-up {EAP} installation version 7.2 or higher is available. The base directory of the {EAP} installation is referred to as `__EAP_HOME__`.
+* {PRODUCT} is installed and configured.
+* Sufficient user permissions to complete the installation are granted.
+
+.Procedure
+. Login to the Red Hat Customer Portal and navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Log in to the...

Actually, delete this:
Login to the Red Hat Customer Portal and
So you have:
Navigate to the ...

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
++
+* *Product:* Data Grid
+* *Version:* 7.3
+
+. Download and unzip the *Red Hat JBoss Data Grid 7.3.0 Server* (`jboss-datagrid-7.3.0-1-server.zip`) installation file to the preferred location on your machine.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Don't use 'machine'. Use system or computer or something else.

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,103 @@
+[id='clustering-quartz-proc_{context}']
+= Configuring Quartz timer service
+
+When you run {KIE_SERVER} in a cluster you must configure the Quartz timer service.
+
+Before you configure a database on your application server, you must prepare the database for Quartz to create Quartz tables, which will hold the timer data, and the Quartz definition file.
+
+.Prerequisites
+
+* A supported non-JTA data source is available. For example, a PostgreSQL data source.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: ..is available, for example a PostgreSQL data source.

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+* The database must be connected to your application server.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: What database? Is this the data source in the preceding prereq? If so , I would roll this up with the previous:
A supported non-JTA data source is connected to your application server, for example a PostgreSQL data source.

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+.Procedure
+
+. Create Quartz tables in your database to allow timer events synchronization using the DDL script for your database.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: to enable timer events to synchronize using...

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
++
+The DDL script is available in the extracted supplementary ZIP archive in `_QUARTZ_HOME_/docs/dbTables`.
+
+. Create the Quartz configuration file `quartz-definition.properties` in the `_JBOSS_HOME_/_MODE_/configuration/` directory and add the following example content:
++
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: + appears in rendered output. Check.
example content:
So if this is example content why would I want to enter it? Do I enter content similar to the following?

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+#============================================================================
+org.quartz.dataSource.managedDS.jndiURL=jboss/datasources/psbrmsDS
+org.quartz.dataSource.notManagedDS.jndiURL=jboss/datasources/quartzNotManagedDS
+# Note the configured data sources that accommodate the two Quartz schemes at the very end of the file.
+----
+endif::DM[]
++
+[IMPORTANT]
+====
+The recommended interval for cluster discovery is 20 seconds and is set in the `org.quartz.jobStore.clusterCheckinInterval` of the `quartz-definition.properties` file. Depending on your set up consider the performance impact and modify the setting as necessary.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: `org.quartz.jobStore.clusterCheckinInterval` attribute if the.

Not crucial, but consider:
Consider the performance impact on your system and modify the settings as necessary.

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
++
+The `org.quartz.jobStore.driverDelegateClass` property that defines the database dialect. If you use Oracle, set it to `org.quartz.impl.jdbcjobstore.oracle.OracleDelegate`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Stray +. Check rendered output. Missing a verb:
The `org.quartz.jobStore.driverDelegateClass` property that defines the database dialect. Probably?:
The `org.quartz.jobStore.driverDelegateClass` property defines the database dialect. 
'Oracle' is to casual. Do you mean Oracle Real Application Clusters (just guessing, but I imagine something like that).

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+====
+
+. Provide the absolute path to your `quartz-definition.properties` file in the `org.quartz.properties` property.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Out of sequence. Check rendered draft.

File: doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc [hunk 10]
```diff
@@ -1,15 +1,15 @@
 [id='clustering-bc-configure-domain-proc']
 = Configuring a {EAP} {EAP_VERSION} domain-mode cluster for {CENTRAL}
-Configure the {CENTRAL} cluster in domain mode with Elasticsearch and the JMS broker resource adapter integrated with {EAP}. By using domain mode with a cluster, you can centrally manage and publish configurations for your servers.
+Configure the {CENTRAL} cluster in domain mode with {DATAGRID} and the JMS broker resource adapter integrated with {EAP}. By using domain mode with a cluster, you can centrally manage and publish configurations of your servers.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 4 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 5 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Will remove "example" and will add explanations and alternatives for 7.5 Jira

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 7 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 8 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Corrected and removed Oracle line.

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc (hunk 9 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-quartz-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+org.quartz.jobStore.clusterCheckinInterval = 20000
+#=========================================================================
+# Configure Datasources
+#=========================================================================
+org.quartz.dataSource.managedDS.jndiURL=jboss/datasources/psbpmsDS
+org.quartz.dataSource.notManagedDS.jndiURL=jboss/datasources/quartzNotManagedDS
+# Note the configured data sources that accommodate the two Quartz schemes at the very end of the file.
+----
+endif::PAM[]
+ifdef::DM[]
```
Comment (mswiderski) [inline doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: quartz is only used by PAM and should not be then mentioned in DM docs

Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-quartz-proc.adoc]: Done, removed the DM content and added PAM constraint

**Pull request merged**